packages/calypso-e2e/src/lib/base-block.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { ElementHandle } from 'playwright';
+
+/**
+ * Base class for asynchronously initializing objects.
+ */
+export class BaseBlock {
```

Comment:
Would this also cover `coblocks` and other types?

Reply:
I'm thinking yes since all CoBlocks' blocks are essentially Gutenberg blocks. We'd never need a separate base class for Gutenberg and CoBlocks blocks. Does that make sense? Having said that, I can now see that we've already used the `Base*` naming convention for the `BaseContainer` class, so let's stay consistent and keep the prefix as-is.

Reply:
Agreed, let's leave it as-is.